src/main/java/duke/task/TaskManager.java

```diff
@@ -0,0 +1,140 @@
+package duke.task;
+
+import duke.command.CommandNotFoundException;
+import duke.command.MainUI;
+
+
+import java.io.FileOutputStream;
+import java.io.PrintStream;
+import java.util.ArrayList;
+
+import java.io.IOException;
+
+import static duke.command.MainUI.taskArrayList;
+
+public class TaskManager {
+    private static final String TODO_COMMAND = "todo";
+    private static final String DEADLINE_COMMAND = "deadline";
+    private static final String EVENT_COMMAND = "event";
+    public static final String DONE_COMMAND = "done";
+    public static final String DELETE_COMMAND = "delete";
+
+    private static final String ADDED_TASK_MESSAGE = "Got it. I have added this task: ";
+    private static final String MARKED_TASK_AS_DONE_MESSAGE = "Nice! I've marked this task as done: ";
```

Comment:
Great job using named constants! It makes the code much easier to understand.

Reply:
Thanks! :)